objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -1217,7 +1222,432 @@ public void testQuery() throws ImageProcessingException, InterruptedException {
         }
 
     }
+    
+    @Test
+    public void testInvalidPreCropping() throws ImageProcessingException, IOException {
+        // Only run test with intended vantiq availability
+        assumeTrue(testAuthToken != null && testVantiqServer != null);
+        
+        Map config = new LinkedHashMap<>();
+        Map preCrop = new LinkedHashMap<>();
+        
+        // The width and height are larger than the original image, should not do any resizing
+        preCrop.put("x", TOP_LEFT_X_COORDINATE);
+        preCrop.put("y", TOP_LEFT_Y_COORDINATE);
+        preCrop.put("w", 1000);
+        preCrop.put("h", 1000);
+
+        config.put("pbFile", PB_FILE);
+        config.put("metaFile", META_FILE);
+        config.put("outputDir", OUTPUT_DIR);
+        config.put("saveRate", SAVE_RATE);
+        config.put("saveImage", "local");
+        config.put("preCrop", preCrop);
+        checkInvalidPreCropping(config);
+        
+        // The coordinates are negative, should not do any resizing
+        preCrop.put("x", -1);
+        preCrop.put("y", -1);
+        preCrop.put("w", CROPPED_WIDTH);
```

Comment:
may be the same code, but good to check these individually?  Also, check for 0 h & w, as well as missing (only, x, x & y, x&y&h, etc.